contrib/git/packages.el

```diff
@@ -229,7 +229,8 @@ which require an initialization must be listed explicitly in the list.")
         "gb" 'magit-blame-mode
         "gl" 'magit-log
         "gs" 'magit-status
-        "gC" 'magit-commit)
+        "gC" 'magit-commit
+        "gp" 'magit-push)
```

Comment:
These action is pretty dangerous, seems that there is no prompt to confirm the action. So I won't set a key binding for this.